app/views/support_interface/performance/index.html.erb

```diff
@@ -1,8 +1,10 @@
 <% content_for :title, 'Service performance dashboard' %>
```

Comment:
Here's an alternative design: https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/commit/7c462bddd3e3ef9497a75107fc4fb432b0c09ec0

![image](https://user-images.githubusercontent.com/233676/73343323-53f07e00-4278-11ea-9b1f-0bb9146454c0.png)

Until we have Apply 2, candidates and application forms are essentially the same. This version removes the duplication of "unsubmitted" applications, and uses the blocks. Once https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/1201 is merged we can link to that for the explanation of the state.

Reply:
Cleaner, but I think harder to interpret at a glance. There's no sense of progression with this one.

Reply:
Can we stick with the original design but apply the deduplication?

Reply:
Isn't it the same sense of progression (left to right) as the existing blocks?

Perhaps "Ended without success" and "Enrolled" should be pulled out because they're end states.

Reply:
In follow-up work, I'd be keen to further break down the data by cohorts, based on when users signed up. Here's a mock-up:

![image](https://user-images.githubusercontent.com/23801/73383647-9ab69600-42c1-11ea-95e0-ce9890536cbd.png)

This breakdown would allow us to see:

- how our product improvements are playing out over time – hopefully we should see better conversions as we improve stuff
- see a more accurate conversion rate i.e. more recent applications, where the candidate hasn't had time to submit it yet, won't distort the completion rate for older applications